### PR TITLE
Add custom events and properties support with latest native SDKs

### PR DESCRIPTION
**Pull request by ranterle**
Adds support for custom events and custom properties with the latest native SDKs.

### CODE REVIEW

**Comment by Cheesebaron** [comment]
Yes! Looking very much forward to test this one out.

File: README.md [hunk 1]
```diff
@@ -73,25 +78,25 @@ Please see the "[How to create a new app](http://support.hockeyapp.net/kb/about-
...
 <a id="integrate-sdk"></a>
 ### 2.2 Integrate the SDK
-For each iOS and Android project desired, add the HockeySDK-Xamarin nuget.
+For each iOS and Android project desired, add the HockeySDK-Xamarin nuget package.
 
 #### For Xamarin Studio
-1. Navigate `Project->Add NuGet Packages...`
-2. Search `HockeySDK.Xamarin` (enable show pre-release packages)
+1. Navigate to `Project -> Add NuGet Packages...`
+2. Search for `HockeySDK.Xamarin` (enable pre-release packages)
```
Comment (DangMH) [inline README.md]: Isn't 4.1.0 no longer pre-release?

File: README.md [hunk 2]
```diff
@@ -73,25 +78,25 @@ Please see the "[How to create a new app](http://support.hockeyapp.net/kb/about-
...
 
 #### For Xamarin for Visual Studio
-1. Navigate `Project->Manage NuGet Packages...`
+1. Navigate `Project -> Manage NuGet Packages...`
 2. Search `HockeySDK.Xamarin` (enable include prerelease)
```
Comment (DangMH) [inline README.md]: same as above

File: README.md (hunk 1 above)
Comment (ranterle) [inline README.md]: Correct.

File: README.md (hunk 2 above)
Comment (ranterle) [inline README.md]: Thanks, will change.

**Pull request merged**